spec/javascripts/govuk_publishing_components/lib/trigger-event-spec.js

```diff
@@ -8,6 +8,9 @@ describe('The trigger event code', function () {
       if (((event || {}).detail || {}).test === true) {
         obj.secondCalledFunction()
       }
+      if (event.keyCode === 13) {
```

Comment:
What does 13 signify? Can we assign the value to a variable name that describes that please?

Reply:
For the purposes of the test it doesn't matter what gets passed - the test passes a keycode to the triggerEvent script, then checks that the same keycode comes out the other end in the called function.

If you're still interested though it's the keycode for Enter.

Reply:
Yeah I know, and the tests explain what's going on well but this bit made me think ([Don't Make Me Think](https://www.interaction-design.org/literature/article/don-t-make-me-think-key-learning-points-for-ux-design-for-the-web)). It just makes it easier to read. You could add a comment?

```suggestion
      // For example if user presses Enter
      if (event.keyCode === 13) {
```

```suggestion
      enterKeyCode = 13
      if (event.keyCode === enterKeyCode) {
```

Reply:
I thought about it, but I think it detracts from the point of the test. I could have put anything in here, it's not significant.

Reply:
It's nbd as it was a suggestion and not a blocker anyway.